internal/resolution/lockfile/npm_test.go

```diff
@@ -0,0 +1,143 @@
+package lockfile_test
+
+import (
+	"testing"
+
+	"deps.dev/util/resolve"
+	"deps.dev/util/resolve/dep"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/osv-scanner/internal/resolution/lockfile"
+	lf "github.com/google/osv-scanner/pkg/lockfile"
+)
+
+func npmVK(name, version string) resolve.VersionKey {
```

Comment:
nit: this should ideally use `t.Helper()` methinks

Reply:
What does `t.Helper()` actually do?
The function just makes a struct so I didn't really want to pass in the `*testing.T` if I didn't have to.

Reply:
> Helper marks the calling function as a test helper function. When printing file and line information, that function will be skipped

While I agree it's a bit annoying, I'd say "that's Go for you 🙃" - to me _because_ this function "just makes a struct" that (unfortunately) means it really should be marked with `t.Helper()` because you want it to be a setup detail of the test rather than the test itself.

There's also the consistency side - we're using it pretty much everywhere else already.

Reply:
Ack.
I'll add it here (and also to the similar function in the npm manifest tests)